policyengine_us/variables/gov/states/oh/tax/income/credits/oh_senior_citizen_credits.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class oh_senior_citizen_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio senior citizen credits"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=20",
+        "https://codes.ohio.gov/ohio-revised-code/section-5747.055",
+    )
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        age_threshold = parameters(
```

Comment:
Then use `p.age_threshold` below directly - same for other parameters
```suggestion
        p = parameters(period).gov.states.oh.tax.income.credits.senior_citizen
```